Bootstrap: add key names option and do a small refactor (#277)
repository-service-tuf/repository-service-tuf-cli: docs/source/guide/index.rst

    Choose 1/1 ONLINE key type [ed25519/ecdsa/rsa] (ed25519):
    Enter 1/1 the ONLINE`s private key path: tests/files/online.key
    Enter 1/1 the ONLINE`s private key password:

>     You can give a name to the key to have a reference in future:

Reviewer: We could use something shorter and more straightforward for this sentence.
```suggestion
    [Optional] Give a name/tag to the key:
```

I'm suggesting “tag” because it is something commonly used nowadays.

    Choose 1/2 root key type [ed25519/ecdsa/rsa] (ed25519):
    Enter 1/2 the root`s private key path: tests/files/JanisJoplin.key
    Enter 1/2 the root`s private key password:

>     You can give a name to the key to have a reference in future:

Reviewer: ```suggestion
    [Optional] Give a name/tag to the key:
```

    ✅ Key 1/2 Verified

    Choose 2/2 root key type [ed25519/ecdsa/rsa] (ed25519):
    Enter 2/2 the root`s private key path: tests/files/JimiHendrix.key
    Enter 2/2 the root`s private key password:

>     You can give a name to the key to have a reference in future:

Reviewer: ```suggestion
    [Optional] Give a name/tag to the key:
```
